### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by nezia1**
This is a really trivial documentation update fixing some typos, and overall making it a bit more readable.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -77,10 +76,10 @@ The structure for smoke tests is explained in the smoke testing package: [`packa
 
 ## Technical Debt
 
-This is just brief summary of Insomnia's current technical debt.
+This is just a brief summary of Insomnia's current technical debt.
 
 - Loading large responses (~20MB) can crash the app on weaker hardware.
-- Redux stores an in-memory duplicate of the local DB, unnecessarily doubling memory usage. Moving
+- Redux stores are an in-memory duplicate of the local DB, unnecessarily doubling memory usage. Moving
```
Comment (develohpanda) [inline DEVELOPMENT.md]: The previous version is correct here, but might be worth rewording? `An in-memory duplicate of the local DB is stored in Redux, unnecessarily doubling...`

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -4,31 +4,30 @@ The purpose of this document is to provide a general overview of the application
...
+- [`styled-components`](https://styled-components.com/) and [`Less`](http://lesscss.org/) are used
   for styling UI components.
 - [`Electron Builder`](https://github.com/electron-userland/electron-builder) is used to help build,
   sign, and package Insomnia for distribution.
 - [`Flow`](https://flow.org/) is used for adding types to the codebase. Not everything is Flow but
   all new code should be typed with Flow.
 - [`Webpack`](https://webpack.js.org/) is the bundler used to compile the JS/Less/babel/etc
-  libcurl library that Insomnia uses to make requests. Libcurl is the HTTP client of choice because
-  it allows the deepest amount of debuggability and control of HTTP requests.
+- [`libcurl`](https://curl.se/libcurl/) is the library that Insomnia uses to make requests. Libcurl is the HTTP client of choice because it allows the deepest amount of debuggability and control of HTTP requests.
```
Comment (develohpanda) [inline DEVELOPMENT.md:21]: Nice catch 😅

**Review by develohpanda — CHANGES_REQUESTED** [review]
Thanks for raising this PR, and welcome to the project! One small change, please 🙂

File: DEVELOPMENT.md (hunk 1 above)
Comment (Nezia1) [reply DEVELOPMENT.md]: Sorry about that, I got really confused about the use of "stores" here, I thought the sentence was mentioning the Redux stores. Given that it confused me, I rephrased it a bit, I think it makes it a bit easier to understand?

Comment (nezia1) [reply DEVELOPMENT.md]: Sorry about that, I got really confused about the use of "stores" here, I thought the sentence was mentioning the Redux stores. My bad 😅

**Review by develohpanda — APPROVED** [review]

File: DEVELOPMENT.md (hunk 1 above)
Comment (develohpanda) [reply DEVELOPMENT.md]: I had to do a double-take as well 😆

**Pull request merged**